sympy/tensor/array/tests/test_ndim_array.py

```diff
@@ -0,0 +1,31 @@
+from sympy.utilities.pytest import raises
+from sympy import (Array, ImmutableDenseNDimArray, ImmutableSparseNDimArray,
+    MutableDenseNDimArray, MutableSparseNDimArray)
+
+
+array_types = [
+    ImmutableDenseNDimArray,
+    ImmutableSparseNDimArray,
+    MutableDenseNDimArray,
+    MutableSparseNDimArray
+]
+
+
+def test_array_negative_indices():
+    for ArrayType in array_types:
+        test_array = ArrayType([[1, 2, 3, 4, 5], [6, 7, 8, 9, 10]])
+        assert test_array[:, -1] == Array([5, 10])
+        assert test_array[:, -2] == Array([4, 9])
+        assert test_array[:, -3] == Array([3, 8])
+        assert test_array[:, -4] == Array([2, 7])
+        assert test_array[:, -5] == Array([1, 6])
+        assert test_array[:, 0] == Array([1, 6])
+        assert test_array[:, 1] == Array([2, 7])
+        assert test_array[:, 2] == Array([3, 8])
+        assert test_array[:, 3] == Array([4, 9])
+        assert test_array[:, 4] == Array([5, 10])
+
+        assert raises(ValueError, lambda: test_array[:, -6])
```

Comment:
This test is failing.

Reply:
Yeah, `raises` should be used without `assert`.